src/shared/components/VideoPlayer/videoJsPlayer.ts

```diff
@@ -3,16 +3,16 @@ import videojs, { VideoJsPlayer, VideoJsPlayerOptions } from 'video.js'
 import 'video.js/dist/video-js.css'
 
 import { STORAGE_NODE_URL } from '@/config/urls'
-import { MediaLocation } from '@/api/queries'
+import { Asset, AssetStorage, VideoFieldsFragment } from '@/api/queries'
 
 export type VideoJsConfig = {
-  src: MediaLocation
+  src: VideoFieldsFragment['media']
```

Comment:
I think it can accept a string now and we can do the resolving outside, just like we do for the `posterUrl`